src/listproperties/listpropertiesediting.ts

```diff
@@ -312,13 +336,18 @@ function createAttributeStrategies( enabledProperties: ListPropertiesConfig ) {
 
 						return;
 					}
+				} else {
```

Comment:
This looks duplicate to the code below. I would assume that this code should remove attribute (or whatever previous code did)

Reply:
removed